conformity/fields/geo.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import absolute_import, unicode_literals
 
 import attr
-
```

Comment:
Please undo this line removal. The `attr` import should be in a separate group from the `conformity` import.